rmgpy/molecule/draw.py

```diff
@@ -1041,12 +1046,17 @@ def _draw_line(self, cr, x1, y1, x2, y2, dashed=False):
         """
         Draw a line on the given Cairo context `cr` from (`x1`, `y1`) to
         (`x2`,`y2`), and update the bounding rectangle if necessary.
+
+        For a dashed line, `dashed` be a list of on/off segment lengths,
+        or `True` in which case [3.5,3.5] is used.
```

Comment:
Minor: please use double "`" marks for `True`, a single mark on each side represents a link in rst (double, a code)